.prettierrc

```
@@ -0,0 +1,7 @@
+{
+  "arrowParens": "always",
+  "semi": true,
+  "singleQuote": true,
+  "tabWidth": 2,
```

The codebase is already formatted with tab width 4. Keeping that will significantly reduce the number of changes you're trying to make all at once.
```suggestion
  "tabWidth": 4,
```